### PR TITLE
Rocket X

### PR DESCRIPTION
**Pull request by Mahak1729**
Created File , Created Rocket X Code

### CODE REVIEW

File: a03_Mahak1729.py [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+# Purpose: TO BUILD COOL STUFF AND LEARN FUNCTIONS
+# Google Doc Link: https://docs.google.com/document/d/1XCSB1JgXj2vmlOMzZ0txzvQJ5EK_cw7_2M5RI1vVwaM/edit]\
+#################################################################################
+# Acknowledgements:
+# Used My own code for starter code from HW02
+#################################################################################
+import turtle
+
+
+def sml_circle(shape):
```
Comment (sheggen) [inline a03_Mahak1729.py:14]: All functions should include a docstring.

File: a03_Mahak1729.py [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+    turtle.colormode(255)  # Gives a color range to choose the rgb from / limits to
+    wn.bgcolor('#ffb06d')  # Sets background color to light orange
+
+    shape = turtle.Turtle()  # Initiates turtle shape
+    shape.width(2)
+    shape.color('#B0BEC5')  # Sets initial turtle color to light grayish blue
+    shape.begin_fill()  # Fills with color
+    for side in range(3):  # Makes triangle(s)
+        shape.forward(100)
+        shape.left(120)
```
Comment (sheggen) [inline a03_Mahak1729.py:45]: Why not encapsulate into a function?

File: a03_Mahak1729.py [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+
+    shape.color('#CFD8DC')  # Sets triangle turtle color to light bluish gray
+    shape.pendown()
+    shape.width(2)
+    shape.begin_fill()
+    for side in range(2):
+        shape.forward(100)
+        shape.right(90)
+        shape.forward(140)
+        shape.right(90)
```
Comment (sheggen) [inline a03_Mahak1729.py:57]: Another function?

File: a03_Mahak1729.py [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+    shape.forward(30)
+    shape.left(50)
+    shape.forward(30)
+    shape.right(125)
+    shape.forward(30)
+    shape.left(50)
+    shape.forward(30)
+    shape.right(125)
+    shape.forward(45)
+    shape.penup()
```
Comment (sheggen) [inline a03_Mahak1729.py:83]: Another function!

Comment (Mahak1729) [reply a03_Mahak1729.py:73]: I made the proper changes in the latest commit now for all the code/errors in it.

**Pull request merged**